Use MAVSDK queue_message instead of send_message (fix #79)

marvind opened the pull request:
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Fixes all warnings we currently get when building RCCar and ControlTower coming from using the deprecated send_message interface of MAVSDK. Fixes #79 .


* **What is the new behavior (if this is a feature change)?**
No changed functionality but needed to adapt some code to use queue_message instead of send_message.


* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No.


* **Other information**:
Raises minimum version of MAVSDK, because queue_message needs to be present. When using MAVSDK 2+ and not earlier dev versions, you should be fine.
@ariamirzai adding you as reviewer as I mainly touched your code for getting route from vehicle and logging, could you double check whether there is no odd behavior?

marvind committed:
MavsdkVehicleServer: use MAVSDK queue_message instead of deprecated send_message

marvind committed:
MavsdkVehicleConnection: use MAVSDK queue_message instead of deprecated send_message

marvind committed:
PlanUI: fix "maybe unitialized" warning when building Release

ariamirzai reviewed (changes requested):
**- PlanUI: Imported mission from the vehicle does not appear in the correct place.**

- Downloading a mission from the vehicle worked fine.
- The logger seems worked fine.

marvind committed:
PlanUI: fix reading of ENU ref from route XML

marvind commented:
> **- PlanUI: Imported mission from the vehicle does not appear in the correct place.**

Fixed in latest commit. Was actually an existing bug that was hidden because `llh_t importedEnuRef` was not initialized. The bug was triggered because I fixed a compiler warning initializing it.

ariamirzai reviewed (approved):
Tested and works!
